unit/testing-utils/load_java_class.h

```diff
@@ -19,11 +19,13 @@
 
 symbol_tablet load_java_class(
   const std::string &java_class_name,
-  const std::string &class_path);
+  const std::string &class_path,
+  const std::string &main = "");
```

Comment:
This is a sensible default because `config.main ==  ""` => use the `main` method?

Reply:
Yes, this is now indicated in the documentation.